doc/nrf/gs_installing.rst

```diff
@@ -316,38 +316,58 @@ To switch to the latest state of development, enter the following commands::
    Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
    This will bring the project repositories to the matching revision defined by the manifest file.
 
-Updating your existing clones to use west
-=========================================
+.. _repo_move:
 
-If you already cloned the |NCS| repositories in Git and want to continue using these clones instead of creating new ones, you can initialize west to use your clones.
-All branches, remotes, and other configuration in your repositories will be maintained.
+Pointing the repositories to the right remotes after they were moved
+********************************************************************
 
-To update your repositories to be managed by west, make sure that they are structured and named in the following way::
+Short before |NCS| version 1.3.0 was released, the full set of repositories were moved from the ``NordicPlayground`` GiHub organization to their current home, the ``nrfconnect`` organization. They were also renamed, replacing the ``fw-nrfconnect-`` prefix with ``sdk-``.
```

Comment:
```suggestion
Shortly before |NCS| version 1.3.0 was released, the Git repositories were moved from the ``NordicPlayground`` GitHub organization to their current home, the ``nrfconnect`` organization. They were also renamed, replacing the ``fw-nrfconnect-`` prefix with ``sdk-``.
```

(The "set" removal is because otherwise it's the subject, which means the verbs would have to be singular IMO.)